### ISSUE TITLE
Exception: Cannot import inventory without organization root

### ISSUE THREAD

**Issue by jceresini** [GoogleCloudPlatform/forseti-security#1614]
Using forseti 2.0.0-rc2, and configuring the server to run with a project as the "root resource", inventory works as expected, but I get the following error when trying to model it:

```
2018-05-29 19:21:57,533 ERROR google.cloud.forseti.services.dao(set_error): warning = , message = Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/forseti_security-2.0.0-py2.7.egg/google/cloud/forseti/services/model/importer/importer.py", line 200, in run
    'Cannot import inventory without organization root')
Exception: Cannot import inventory without organization root
```

It seems odd that I'm able to specify a project or folder as the root resource, but I can only run inventory with that configuration. Looking at the file mentioned above, I'm not sure why the conditional/exception is needed:

```
                if root.get_resource_type() in ['organization']:
                    self.found_root = True
                if not self.found_root:
                    raise Exception(
                        'Cannot import inventory without organization root')
```

The `_convert_project` method checks to see if the project is the root, which doesn't make sense if thats not allowed:

```
    def _convert_project(self, project):
        """Convert a project to a database object.

        Args:
            project (object): Project to store.
        """

        data = project.get_resource_data()
        if self._is_root(project):
            parent, type_name = None, self._type_name(project)
            full_res_name = type_name
        else:
            parent, full_res_name, type_name = self._full_resource_name(
                project)
```

Is there another reason for this exception, or is it something that can be removed. We have some use cases where we'd like to inventory, model, scan, etc on a per-project basis.

**Comment by joecheuk** [comment]
Hi @jceresini , can you try running again on branch "2.0-rc3"? We added some handles to not throw exception and continue when service account doesn't have org access.

**Comment by jceresini** [comment]
In this case its not a matter of having access to the org. I'm setting the root_resource_id to a project so it doesn't event attempt to access the org. The same logic that raises an exception for a non-org root_resource is present in 2.0-rc3:

https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-rc3/google/cloud/forseti/services/model/importer/importer.py

```
                if root.get_resource_type() in ['organization']:
                    LOGGER.debug('Root resource is organization: %s', root)
                    self.found_root = True
                if not self.found_root:
                    LOGGER.debug('Root resource is not organization: %s.', root)
                    raise Exception(
                        'Cannot import inventory without organization root')

```

**Comment by jceresini** [comment]
An additional note, if I comment out the exception, inventory proceeds to load all of the project's inventory. So given that the server config allows non-org roots, and the fact that inventory collects the project resources, it seems odd to throw an exception and give up when you try to do it.

**Comment by ocervell** [comment]
Still happening for me in latest `dev` branch. Commenting out the exception fixes the issue.

**Comment by rvandegrift** [comment]
I thought this was fixed a while back - @jceresini, are you aware of any existing issues with `root_resource_id` not being an organization?

My "sandbox" forseti installation is 2.8.0, and `root_resource_id` is pointed to a project.  Inventory, model, and scanner are all working fine.

**Comment by joecheuk** [comment]
Resolved #1770
